modules/datagrid/72/launch/added/launch/infinispan-config.sh

```diff
@@ -489,10 +520,18 @@ function configure_jdbc_store() {
           ;;
       esac
     fi
-    if [ -n "$(find_env "${prefix}_CACHE_EVICTION_STRATEGY")" -a "$(find_env "${prefix}_CACHE_EVICTION_STRATEGY")" != "NONE" ]; then
-      JDBC_STORE_PASSIVATION=true
-    else
-      JDBC_STORE_PASSIVATION=false
+    if [ -n "$(find_env "${prefix}_CACHE_MEMORY_EVICTION_SIZE")$(find_env "${prefix}_CACHE_EVICTION_MAX_ENTRIES")" ]; then
```

Comment:
@vblagoje @osnipso This seems to break backward compatibility... previously cachestore passivation depended on '_CACHE_EVICTION_STRATEGY' and now it depends on '_CACHE_MEMORY_EVICTION_SIZE' and '_CACHE_EVICTION_MAX_ENTRIES'.  But that could be  considered as a fix if passivation itself didn't take any effect without having eviction size set up. One way or another, this change should be mentioned in release notes for a new stream since it changes/fixes behaviour .